apps/files/src/mixins.js

```diff
@@ -11,11 +11,16 @@ export default {
       if (int < 0) {
         return ''
       }
+
       if (isNaN(int)) {
         return '?'
       }
+
+      const mb = 1048576
+
+      // TODO: Pass current language as locale to display correct separator
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat

not sure if quickly doable ?

Reply:
please raise issue to address this

Reply:
Filesize supports locale on its own - https://www.npmjs.com/package/filesize

This is more about figuring out a nice way to get there the locale.

Reply:
Issue raised - #3067